resources/views/components/package.blade.php

```diff
@@ -62,8 +62,8 @@
             }}
         </h3>
         <package-stats :package="$package" />
-        <p>{{ $package['description'] }}</p>
-        <a-styled :href="data_get($package, 'repository')" underlined class="mt-4">
+        <p class="flex-grow">{{ $package['description'] }}</p>
+        <a-styled :href="data_get($package, 'repository')" underlined class="mt-4 w-max">
```

Comment:
I think that a `w-full` would work as well!? Or even simpler the class `block`. 🤔